### DISCUSSION TITLE
Add additional informations to the structure pane

### DISCUSSION THREAD

**Discussion by spechto** [texstudio-org/texstudio#3715] [Q&A]
Hello,

at the moment I am using WinEdt with MikTeX and I have played a bit with texstudio and found the image preview very helpful. And especially the ability to paste png files from the clipboard. Very nice!

I also saw the structure pane. Is there any way to add additional information to it? \todo commands are recognised, it would be great to add own sections for example \bibitem or other (self defined) latex commands.

Best regards 
Martin

**Comment by sunderme** [comment]
Please look at the user manual: https://texstudio-org.github.io/editing.html#the-structure-view
There are no other mechanisms as described there.

**Comment by spechto** [comment]
It looks as if there are no further options for adding more informations to the structure view. So I'll pass this on to the ideas section.
